### PR TITLE
fix: Exclude color contrast test results (false positives)

### PR DESCRIPTION
**Pull request by DaveTryon**
#### Description of changes
We've had reports of false positives in the color contrasts test, so we're temporarily excluding the results from the output. This is intended to be a short-term change that will be reverted after we are able to improve the error rate of the results being returned from axe-android.

#### Pull request checklist

- [n/a] Addresses an existing issue: #0000
- [x] Ran `yarn fastpass`
- [x] Added/updated relevant unit test(s) (and ran `yarn test`)
- [x] Verified code coverage for the changes made. Check coverage report at: `<rootDir>/test-results/unit/coverage`
- [x] PR title *AND* final merge commit title both start with a semantic tag (`fix:`, `chore:`, `feat(feature-name):`, `refactor:`). See `CONTRIBUTING.md`.
- [n/a] (UI changes only) Added screenshots/GIFs to description above
- [n/a] (UI changes only) Verified usability with NVDA/JAWS

### CODE REVIEW

File: src/tests/unit/tests/common/components/cards/__snapshots__/rule-resources.test.tsx.snap [hunk 1]
```diff
@@ -3,7 +3,7 @@
 exports[`RuleResources renders with {
   url: 'test-url',
   guidanceLinks: [ [length]: 0 ],
```
Comment (DaveTryon) [inline src/tests/unit/tests/common/components/cards/__snapshots__/rule-resources.test.tsx.snap:5]: This file needed to be updated to pass on my machine. Not really sure why, since it seems unrelated to these changes

Comment (peterdur) [reply src/tests/unit/tests/common/components/cards/__snapshots__/rule-resources.test.tsx.snap:5]: Looks like the PR build unit tests are failing here. When I tried out this change on my machine, running unit tests generated new snapshots without the `(anonymous)`. I'm guessing something is out of sync? Was a `yarn install` missed, or is your `node --version` something other than `v12.16.3`?

**Comment by DaveTryon** [comment]
Pausing this until I get the details sorted out

File: src/tests/unit/tests/common/components/cards/__snapshots__/rule-resources.test.tsx.snap (hunk 1 above)
Comment (DaveTryon) [reply src/tests/unit/tests/common/components/cards/__snapshots__/rule-resources.test.tsx.snap:5]: Node version issue. Homebrew installed 14.X on my mac. It works now that I forced it back to 12.X
